ReflectionAnalyzers.Vsix/source.extension.vsixmanifest

```diff
@@ -1,30 +1,24 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
-    <Identity Id="ReflectionAnalyzers.Vsix.891942ac-38e5-4220-b8d7-a96617a1bd07" Version="1.0" Language="en-US" Publisher="Johan Larsson" />
-    <DisplayName>ReflectionAnalyzers.Vsix</DisplayName>
+    <Identity Id="ReflectionAnalyzers.891942ac-38e5-4220-b8d7-a96617a1bd07" Version="0.1.20" Language="en-US" Publisher="Johan Larsson" />
```

Comment:
📝 This can be pulled from the project.

```
Version="|%CurrentProject%;GetVsixVersion|"
```

Then in the project file:

```xml
  <Target Name="GetVsixVersion" Returns="$(VsixVersion)">
    <PropertyGroup>
      <VsixVersion>0.1.20</VsixVersion>
    </PropertyGroup>
  </Target>
```

You can then modify this to set `$(VsixVersion)` according to the same property you use to update the package version (via whatever method you use for versioning in the project).

Reply:
Ooh this is way better!

Reply:
Nice skills Sam!